scanning/scanners/pipeline_scanner.py

```diff
@@ -53,7 +53,8 @@ def process_output(self, json_data):
             for update in logs.get("Scan Results").get("Package Updates"):
                 msg = msg + update + "\n"
             data["msg"] = msg
-            data["alert"] = True
+            # We are not counting RPM updates as alerts for now
+            data["alert"] = False
```

Comment:
package update remaining is also a problem IMO. may be we can re-iterate on this later